### PR TITLE
feat(templates/sotd.html): link to 2017 patent policy

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
I checked the fragments, and they are still good in the new link.

**Comment by marcoscaceres** [comment]
Question: should it say "5 February 2004 (updated 1 August 2017)" or just leave it as "5 February 2004"?

**Comment by marcoscaceres** [comment]
Thanks for the heads-up on this... maybe of interest to @anssiko, who has the WICG GeoSensor proposal.

**Comment by marcoscaceres** [comment]
(sorry about the geo comment... wrong bug!)

File: js/w3c/templates/sotd.html [hunk 1]
```diff
@@ -131,7 +131,7 @@
             a group
             {{/if}} operating under the
             <a{{#if doRDFa}} id="sotd_patent" property='w3p:patentRules'{{/if}}
-            href='https://www.w3.org/Consortium/Patent-Policy-20040205/'>5 February 2004 W3C Patent
+            href='https://www.w3.org/Consortium/Patent-Policy-20170801/'>5 February 2004 W3C Patent
```
Comment (anssiko) [inline js/w3c/templates/sotd.html]: Should this use the Patent Policy's full name:

`5 February 2004 (updated 1 August 2017)`

@plehegar to give an authoritative answer.

**Comment by deniak** [comment]
> Question: should it say "5 February 2004 (updated 1 August 2017)" or just leave it as "5 February 2004"?
In the https://github.com/w3c/specberus/pull/697, I replaced "5 February 2004" with "1 August 2017". @plehegar what do you think?

**Comment by plehegar** [comment]
(I need to check with Wendy)

**Comment by wseltzer** [comment]
Please use "5 February 2004 (updated 1 August 2017)"

**Comment by marcoscaceres** [comment]
@wseltzer thanks! done :)

File: js/w3c/templates/sotd.html [hunk 2]
```diff
@@ -131,8 +131,9 @@
             a group
             {{/if}} operating under the
             <a{{#if doRDFa}} id="sotd_patent" property='w3p:patentRules'{{/if}}
-            href='https://www.w3.org/Consortium/Patent-Policy-20040205/'>5 February 2004 W3C Patent
-            Policy</a>.
+            href='https://www.w3.org/Consortium/Patent-Policy-20170801/'>
+            5 February 2004 (updated 1 August 2017)
```
Comment (tidoust) [inline js/w3c/templates/sotd.html]: "W3C Patent Policy" seems to have been swallowed in the update! Text should "5 February 2004 (updated 1 August 2017) W3C Patent Policy"

**Comment by tidoust** [comment]
Actually, note the updated text that made it to pubrules no longer references the publication date and rather uses undated URLs, so this PR needs some update:
https://github.com/w3c/specberus/pull/697#issuecomment-336212734

File: js/w3c/templates/sotd.html (hunk 2 above)
Comment (marcoscaceres) [reply js/w3c/templates/sotd.html]: Well spotted, thanks!

**Pull request merged**